kernel/shared/src/main/scala/monocle/function/Possible.scala

```diff
@@ -0,0 +1,24 @@
+package monocle.function
+
+import monocle.Optional
+
+trait Possible[S] {
+  type A
+  def possible: Optional[S, A]
+}
+
+object Possible {
+  type Aux[S, A0] = Possible[S] { type A = A0 }
+
+  def apply[S, A0](optional: Optional[S, A0]): Aux[S, A0] = new Possible[S] {
+    type A = A0
+    override val possible: Optional[S, A0] = optional
+  }
+
+  implicit def optionPossible[A0](implicit ev: Optional[Option[A0], A0]): Possible[Option[A0]] =
```

Comment:
I guess we don't want this `implicit` do we?

Reply:
yeah here we should define an instance of the typeclass Possible for `Option[A]`. So the signature is most likely going to be:
```scala
implicit def option[A]: Possible.Aux[Option[A], A] = apply(...) 
```